Formula/easy-rsa.rb

```diff
@@ -0,0 +1,34 @@
+class EasyRsa < Formula
+  desc "CLI utility to build and manage a PKI CA."
+  homepage "https://github.com/OpenVPN/easy-rsa"
+  url "https://github.com/OpenVPN/easy-rsa/archive/3.0.1.tar.gz"
+  sha256 "a1fff75a27ea7da3f37fbfed715633f55b9ca25f5b14cac38e525c5c995e68ae"
+
+  depends_on "openssl"
+
+  def install
+    # We don't use the installer from the package, as it doesn't organize things
+    # well for brew install. It puts the binary in the prefix root, and defaults
+    # to a pki path that will be overwritten on installation. Instead, I rewrite
+    # its defaults to find its SSL and PKI in brew-appropriate locations.
+    inreplace "easyrsa3/easyrsa" do |s|
+      s.gsub! /set_var EASYRSA		"\$PWD"/,
+              "set_var EASYRSA		\"#{libexec}\""
+      s.gsub! /set_var EASYRSA_PKI	.*/,
+              "set_var EASYRSA_PKI	\"#{etc}/pki\""
+      openssl_path = Formula["openssl"].opt_bin/"openssl"
+      ohai "Using OpenSSL from #{openssl_path}"
```

Comment:
Let's not `ohai`. It's not really useful knowledge given the dependency makes it fairly obvious.